temporal-opentracing/src/main/java/io/temporal/opentracing/StartSpanContext.java

```diff
@@ -0,0 +1,76 @@
+/*
+ *  Copyright (C) 2020 Temporal Technologies, Inc. All Rights Reserved.
+ *
+ *  Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ *  use this file except in compliance with the License. A copy of the License is
+ *  located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package io.temporal.opentracing;
+
+public class StartSpanContext {
+
+  private OpenTracingOptions options;
+  private SpanOperationType spanOperationType;
+  private String typeName;
+  private String workflowId;
+  private String runId;
+  private String parentWorkflowId;
+  private String parentRunId;
+
+  public StartSpanContext(
+      OpenTracingOptions options,
+      SpanOperationType spanOperationType,
+      String typeName,
+      String workflowId,
+      String runId,
+      String parentWorkflowId,
+      String parentRunId) {
+    this.options = options;
+    this.spanOperationType = spanOperationType;
+    this.typeName = typeName;
```

Comment:
`typeName` is a meaningless name. Yeah, if it's `workflowType` and `activityType` - it's more or less clear what does it mean. But `typeName` is super misleading. Let's invent something like ActionName and add a javadoc clarifying what is expected here (workflowType or activityType)

Reply:
Done - I used `operationName` to go along with `operationType`